### PR TITLE
feat: Add store and forward capability test cases

### PR DESCRIPTION
**Pull request by cherrycl**
Based on #211, writing the related scenarios.

Signed-off-by: Cherry Wang <cherry@iotechsys.com>

### CODE REVIEW

**Review by lenny-intel — DISMISSED** [review]
Nice Job!

File: TAF/testScenarios/integrationTest/UC_end_to_end/export_store_forward.robot [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+StoreAndForward002 - Stored data is cleared after the maximum configured retires
+    Given Set RetryInterval = 3s, MaxRetryCount=3 For app-http-export On Consul
+    And Create Device For device-virtual With Name http-export-device
+    When Get device data by device http-export-device and command ${PREFIX}_GenerateDeviceValue_INT8_RW
+    And Sleep  12s  # wait until retry fails
+    And Start HTTP Server
+    Then HTTP Server Didn't Received Exported Data
+
+StoreAndForward003 - Exporting data didn't retry when Writeable.StoreAndForward.Enabled is false
+    Given Set StoreAndForward.Enable=true, RetryInterval = 1s, MaxRetryCount=3 For app-http-export On Consul
```
Comment (jinlinGuan) [inline TAF/testScenarios/integrationTest/UC_end_to_end/export_store_forward.robot:33]:
```suggestion
    Given Set StoreAndForward.Enable=false, RetryInterval = 1s, MaxRetryCount=3 For app-http-export On Consul
```

**Review by jinlinGuan — CHANGES_REQUESTED** [review]

File: TAF/testScenarios/integrationTest/UC_end_to_end/export_store_forward.robot (hunk 1 above)
Comment (cherrycl) [reply TAF/testScenarios/integrationTest/UC_end_to_end/export_store_forward.robot]: Updated.

**Review by jinlinGuan — APPROVED** [review]
LGTM

**Pull request merged**